### PR TITLE
Page action icon

### PR DESCRIPTION
**Pull request by feedbee**
Add page action icon on tabs that are controlled be the extension. Enable/disable page control on page action icon click. This was discussed under issue #82.

### CODE REVIEW

**Comment by borismus** [comment]
Thanks, this looks good. Could you also make the tooltip change the text, to something like "Enable media keys for this tab" and "Disable media keys for this tab"?

File: extension/background.js [hunk 1]
```diff
@@ -11,8 +11,12 @@ chrome.commands.onCommand.addListener(function(command) {
 chrome.runtime.onMessage.addListener(
     function(request, sender, sendResponse) {
         console.log("Received tab message: ", request);
-        if(request.command == "activateTab" && sender.tab && activeTabs.indexOf(sender.tab.id) == -1){
-            activeTabs.push(sender.tab.id);
+        if(request.command == "activateTab" && sender.tab){
```
Comment (borismus) [inline extension/background.js]: nit: please add spaces after if throughout.

Comment (feedbee) [inline extension/background.js]: Did you mean: `if (request.command == "activateTab" && sender.tab) {`? I used the style that was used in your code before me. If you wish to apply standard code style across all of the files of this extension, it should be done in separeate Pull Request. I'll do if after you merge this PR.

Comment (nemchik) [inline extension/background.js]: I thought if was treated like a function and thus you don't (in good form) add spaces.

Comment (borismus) [inline extension/background.js]: Yes, noticed the stylistic issue throughout. Let's stick to the Google JS style guide: google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml

**Pull request merged**